js/localization/globalize/core.js

```diff
@@ -1,26 +1,12 @@
 // eslint-disable-next-line no-restricted-imports
 import Globalize from 'globalize';
 import coreLocalization from '../core';
+import { likelySubtags } from '../cldr-data/likely_subtags';
```

Comment:
I did not find this file in the localization sources generation task. Are you sure it will exist on other machines?